packages/client/src/components/project/results/TestButton.tsx

```diff
@@ -71,9 +69,9 @@ export default function TestButton(props: {
   return (
     <>
       <Collapse in={!props.active}>
-        <Tooltip title="Test">
-          <IconButton onClick={handleClickPrepare}>Test</IconButton>
-        </Tooltip>
+        <Button variant="outlined" color="primary" onClick={handleClickPrepare}>
+          Test
```

Comment:
Do we want the button to read `Test Model` or something that indicates it is not just a placeholder "test" button. Or maybe `View Model`?

Reply:
I like the one word buttons but we can talk about this some other time